src/dtos/http_response.go

```diff
@@ -0,0 +1,58 @@
+package dtos
+
+import (
+	"time"
+
+	"github.com/Real-Dev-Squad/wisee-backend/src/models"
+)
+
+type ErrorResponse struct {
+	Message string      `json:"message"`
+	Detail  interface{} `json:"detail"`
+}
+
+type ResponseDto struct {
+	Message string        `json:"message"`
+	Data    interface{}   `json:"data"`
+	Error   ErrorResponse `json:"error"`
+}
+
+type CreateUpdateGetFormResponseDto struct {
```

Comment:
- will `CreateUpdateFormResponseDto` be a better name here?

Reply:
It will be used for Get operation also, therefore using this, using CreateUpdateFormResponseDto on a GET call API, doesn't sit right with me, what do you think.